.github/workflows/test-bookdown.yml

```diff
@@ -0,0 +1,60 @@
+# Workflow derived from https://github.com/r-lib/actions/tree/master/examples
+# Need help debugging build failures? Start at https://github.com/r-lib/actions#where-to-find-help
+# Test bookdown building on all branches
+name: test-bookdown
+on:
+  push:
+#    paths: '**.Rmd'
+  workflow_dispatch:
+jobs:
+  test-bookdown:
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: r-lib/actions/setup-pandoc@v1
+
+      - uses: r-lib/actions/setup-r@v1
+        with:
+          use-public-rspm: true
+
+      - name: install remotes
+        run: Rscript -e 'install.packages(c("remotes"))'
+
+      - name: Install system dependencies
```

Comment:
In the actions page, under the [annotation section](https://github.com/NOAA-FIMS/collaborative_workflow/actions/runs/1995113084), it says that there is an error when installing system dependencies, but the workflow worked and created bookdown_html. I am not sure if it is something we need to worry about.